src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs

```diff
@@ -200,28 +194,92 @@ private async Task<RoatpFinancialApplicationViewModel> CreateRoatpFinancialAppli
                 return new RoatpFinancialApplicationViewModel();
             }
 
+            var parentCompanySection = await GetParentCompanySection(applicationFromAssessor.ApplicationId);
+            var activelyTradingSection = await GetActivelyTradingSection(applicationFromAssessor.ApplicationId);
+            var organisationTypeSection = await GetOrganisationTypeSection(applicationFromAssessor.ApplicationId);
+            var financialSections = await GetFinancialSections(applicationFromAssessor.ApplicationId);
+
+            return new RoatpFinancialApplicationViewModel(applicationFromAssessor, parentCompanySection, activelyTradingSection, organisationTypeSection, financialSections);
+        }
+
+        private async Task<Section> GetParentCompanySection(Guid applicationId)
+        {
+            const string ParentCompanySectionTitle = "UK ultimate parent company";
+            const string HasParentCompanyQuestionTag = "HasParentCompany";
+            const string HasParentCompanyPageId = "20";
```

Comment:
Just a heads up, in the APR-1275 branch there's a constants file for all the page ids. It's being reused by assessor so makes sense to use it here too

https://github.com/SkillsFundingAgency/das-admin-service/blob/0b8ec5558fe7f5a2aaac895c92e7cc9882e90e94/src/SFA.DAS.AssessorService.ApplyTypes/Roatp/RoatpQnaConstants.cs

Reply:
the gateway-submaster might have an even longer list, so if you lift it, look in there and compare

Reply:
copied across from gateway-submaster :)